src/components/Articles.jsx

```diff
@@ -1,23 +1,35 @@
-import { useEffect, useContext } from "react"
+import { useEffect, useContext, useState } from "react"
 import { fetchArticles } from "../../axios.js"
 import ArticleCard from "./ArticleCard.jsx"
 import { ArticleContext } from "../contexts/ArticleContext.jsx"
 
 export default function Articles (props) {
 
-    const {articles, setArticles} = props
-    const { article, setArticle } = useContext(ArticleContext)
-    console.log(article)
+    const { articles, setArticles } = props
+    const { setArticle } = useContext(ArticleContext)
```

Comment:
Why destructure  setArticle here using context only to pass it down to article card ?
Defeats the point of context, also not sure you really need to use context at all.